content/libraries/php.mdx

```diff
@@ -21,7 +21,7 @@ composer require top-gg/php-sdk
 
 ### Getting Started
 
-This library follows the PSR standards given by PHP. Because of compliance with this, some things are
+This library follows the PSR standards given by PHP, because of compliance with this some things are
 done differently compared to other API wrappers. Below is sample code on how to initialize the API
```

Comment:
I don't think this is grammatically valid. There needs to be a comma after the introductory clause.

Maybe a better way to reword it could be something like:
> This library does certain things differently from other API wrappers because it follows PHP's PSR standards.

Reply:
Fixed